src/Bridge/Symfony/DependencyInjection/Compiler/AdapterCompilerPass.php

```diff
@@ -47,4 +47,9 @@ private function isDoctrineOrmLoaded(ContainerBuilder $container): bool
     {
         return $container->has('doctrine') && $container->has('sonata.doctrine.adapter.doctrine_orm');
     }
+
+    private function isDoctrinePHPCRLoaded(ContainerBuilder $container): bool
+    {
+        return $container->has('doctrine') && $container->has('sonata.doctrine.adapter.doctrine_phpcr');
```

Comment:
```suggestion
        return $container->has('doctrine_phpcr') && $container->has('sonata.doctrine.adapter.doctrine_phpcr');
```

Reply:
Since this was wrong and test were passing, what about to create a new test for this similar to: 

https://github.com/sonata-project/sonata-doctrine-extensions/blob/da8980b5c57aaa62ef6e8f120aa650ed185e1136/tests/Bridge/Symfony/DependencyInjection/Compiler/AdapterCompilerPassTest.php#L57-L75